buildpipeline/DotNet-CoreClr-Trusted-Windows-x86.json

```diff
@@ -6,14 +6,14 @@
       "alwaysRun": false,
       "displayName": "Install Signing Plugin",
       "timeoutInMinutes": 0,
-      "condition": "ne(variables['PB_SignType'], 'oss')",
+      "condition": "and(succeeded(), or(eq(variables['PB_SignType'], 'real'), eq(variables['PB_SignType'], 'test'))) ",
       "task": {
         "id": "30666190-6959-11e5-9f96-f56098202fef",
         "versionSpec": "1.*",
         "definitionType": "task"
       },
       "inputs": {
-        "signType": "real",
+        "signType": "$PB_SignType",
```

Comment:
Are you sure this works? I suspect it doesn't.

Reply:
@weshaggard I saw this done in another build def.  So presumably yes?

Reply:
I kept this as real to be conservative

Reply:
My assumption is that it would at least need to be `$(PB_SignType)`, but I'm not sure even that works.